src/layers/attention/flash_attn_handler.cpp

```diff
@@ -7,8 +7,6 @@
 #include "memory/kv_cache.h"
 #include "models/parameters.h"
 
-DEFINE_bool(use_kv_cache_stream, true, "use separate stream for kv cache");
```

Comment:
somehow, seperate stream would lead to graph capture failure. just remove the tiny optimization.